docs/ai/quickstarts/snippets/chat-with-data/azure-openai/Program.cs

```diff
@@ -31,7 +31,7 @@
         {
             Key=3,
             Name="Microsoft Entra ID",
-            Description="Manage user identities and control access to your apps, data, and resources.."
+            Description="Manage user identities and control access to your apps, data, and resources."
```

Comment:
There should be a single space between each property name and the assignment, space around each ` = `.